static/app/components/replays/replayView.tsx

```diff
@@ -132,6 +132,26 @@ function ReplayView({toggleFullscreen, isLoading}: Props) {
       ) : null}
     </Fragment>
   );
+
+  // In fullscreen mode, ensure tooltips are rendered within the fullscreen container
+  // to prevent them from being clipped by the fullscreen boundary
+  if (isFullscreen) {
+    return (
+      <TooltipContext
+        value={{
+          // When in fullscreen mode, tooltips must be rendered within the fullscreen
+          // container instead of document.body to avoid being clipped. This ensures
+          // that browser/OS icons and size information tooltips remain visible.
+          // See: https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#things_to_keep_in_mind
+          container: document.fullscreenElement,
+        }}
+      >
+        {content}
```

Comment:
i wanna test to see if this causes the replay to mount/unmount when going in/out of fullscreen.

If it's unmouting then playback would be really broken because the iframe would lose it's stuff.

Reply:
ya. this is causing a re-mount. It's starting the replay from `0:00` whenever i switch in or out of fullscreen mode
i thought it would break worse and get corrupted tbh

Reply:
Fixed things by always wrapping things with `<TooltipContext>`, it's unconditional now. 
I also moved the context to surround everything that's going fullscreen, not just the `<ReplayView>`. This means that when you fullscreen from a replay clip everything will work too.